codeowner.md

```diff
@@ -1,50 +1,32 @@
 # OPEA Project Code Owners
 
-These tables list the GitHub IDs of code owners. For a PR review, please contact the corresponding owner.
-- [GenAIExamples](#genaiexamples)
-- [GenAIComps](#genaicomps)
-- [GenAIEval](#genaieval)
-- [GenAIInfra](#genaiinfra)
-- [CICD](#cicd)
-
-## GenAIExamples
-
-| examples | owner  |
-|:-------------:|:-----------:|
-| AudioQnA      | Spycsh      |
-| ChatQnA       | lvliang-intel|
-| CodeGen       | lvliang-intel|
-| CodeTrans     | Spycsh      |
-| DocSum        | Spycsh      |
-| SearchQnA     | letonghan   |
-| Language Translation |letonghan|
-| VisualQnA     | lvliang-intel|
-| Others        | lvliang-intel|
-
-## GenAIComps
-
-| comps | owner  |
-|:-------------:|:-----------:|
-|asr            |Spycsh       |
-|cores          |lvliang-intel|
-|dataprep       |XinyuYe-Intel|
-|embedding      |XuhuiRen     |
-|guardrails     |letonghan    |
-|llms           |lvliang-intel|
-|reranks        |XuhuiRen     |
-|retrievers     |XuhuiRen     |
-|tts            |Spycsh       |
+These tables list the GitHub code owners, as found in the CODEOWNERS file in the
+corresponding OPEA repository. Code owners are responsible for code and
+documentation in a repository. They are automatically requested for
+review when someone opens a pull request (PR) that modifies code or
+documentation that they own.
+
+Select or contact the corresponding area owner for a PR review or questions
+about content within a repository.
+
+<!-- codeowners.txt is generated at docbuild time from the CODEOWNERS files from each repo -->
+```{include} codeowners.txt
+```
+
+-----
 
 ## GenAIEval
 
-lvliang-intel, changwangss, lkk12014402
```

Comment:
please link to CODEOWNER file of each repo

Reply:
They will be automatically brought in during the doc build process (if the CODEOWNERS file exists).